Failure to compile under clojure 1.8 with future.spec dependency included

Hi there. I'm a relative clojure novice attempting to follow the lacinia tutorial and get a minimal graphql server running. I'm using clojure 1.8.
Having defined the schema as per http://lacinia.readthedocs.io/en/latest/tutorial/init-schema.html, when attempting to run the project I get the following error:

`Exception in thread "main" java.lang.ExceptionInInitializerError
	at clojure.main.<clinit>(main.java:20)
Caused by: java.lang.RuntimeException: Unable to resolve symbol: simple-keyword? in this context, compiling:(com/walmartlabs/lacinia/internal_utils.clj:255:19)`

I've added a project dependency for `clojure-future-spec` as per the docs. Have I done something wrong, or is the current documentation out of date?

Thanks!

--- a/src/com/walmartlabs/lacinia/internal_utils.clj
+++ b/src/com/walmartlabs/lacinia/internal_utils.clj
@@ -19,6 +19,9 @@
     [clojure.string :as str])
   (:import (clojure.lang Named)))
 
+(when (-> *clojure-version* :minor (< 9))
+  (require '[clojure.future :refer [simple-keyword? boolean?]]))
+
 (defmacro cond-let
   "A version of `cond` that allows for `:let` terms. There is hope that someday, perhaps
   after Rich Hickey retires, this will make it into clojure.core."

